lib/reducers/create-otp-reducer.js

```diff
@@ -71,6 +71,7 @@ export function getInitialState (userDefinedConfig, initialQuery) {
     transitOperators: [],
     realtimeEffectsDisplayThreshold: 120,
     routingTypes: [],
+    onTimeThresholdDeltaSeconds: 60,
```

Comment:
`example-config.yml` should include a commented-out line for this new config parameter.